### ISSUE TITLE
For the implementation of quantile-based action binning

### ISSUE THREAD

**Issue by Rooshy-yang** [openvla/openvla#184]
Thank you very much for your excellent work! 

I want to confirm that whether the quantile-based action binning described in the paper is integrated into the code?

In the paper, it’s mentioned:
“...we discretize each dimension of the robot actions separately into one of 256 bins. For each action dimension, we set the bin width to uniformly divide the interval between the 1st and 99th quantile of the actions in the training data.”

When reviewing the code (snippet below), I noticed that it currently uses a fixed uniform binning strategy from min_action to max_action without explicitly referencing quantiles or the training data distribution, see openvla/prismatic/vla/action_tokenizer.py:

`self.bins = np.linspace(min_action, max_action, self.n_bins)`

Would you mind pointing me to where this logic is implemented or describing the intended workflow? 

Thank you so much for your time and assistance!

**Comment by kpertsch** [comment]
Thanks for your question! It's a bit convoluted, but we do the normalization from q01, q99 to -1, 1 in the data loader, before the action tokenizer is applied. So in the tokenizer itself we assume that the data is already normalized to range -1, 1 and just apply the binning. (the reason to do it this way is that when you train on x-embodiment data from many different datasets, you need to apply different q01, q99 normalization stats for each dataset (since their action range if potentially very different), so we do it during data loading (bc we apply the same tokenizer to all datasets)

You can check out the code here: https://github.com/openvla/openvla/blob/1b024f242eda833dc8e321953f25cfd5f3d2f76d/prismatic/vla/datasets/rlds/utils/data_utils.py#L76 

We set NormalizationType.BOUNDS_Q99 here: https://github.com/openvla/openvla/blob/1b024f242eda833dc8e321953f25cfd5f3d2f76d/prismatic/vla/datasets/datasets.py#L99
